src/SFA.DAS.AANHub.Domain/Entities/Member.cs

```diff
@@ -1,5 +1,5 @@
-﻿using System.Text.Json.Serialization;
-using SFA.DAS.AANHub.Domain.Common;
+﻿using SFA.DAS.AANHub.Domain.Common;
+using System.Text.Json.Serialization;
```

Comment:
Your VS settings are incorrect, System namespace should be on top 
![image](https://github.com/SkillsFundingAgency/das-aan-hub-api/assets/29398561/c06d7b8b-f5aa-461a-b00d-5cfdf9a29c8b)

Reply:
ok